problem/0014/README.md

```diff
@@ -0,0 +1,39 @@
+# [14. Longest Common Prefix](https://leetcode.com/problems/longest-common-prefix)
+
+Write a function to find the longest common prefix string amongst an array of strings.
+
+If there is no common prefix, return an empty string `""`.
+
+**Example 1**
+
+```text
+Input: strs = ["flower","flow","flight"]
+Output: "fl"
+```
+
+**Example 2**
+
+```text
+Input: strs = ["dog","racecar","car"]
+Output: ""
+Explanation: There is no common prefix among the input strings.
+```
+
+**Constraints**
+
+```text
+1 <= strs.length <= 200
+0 <= strs[i].length <= 200
+strs[i] consists of only lower-case English letters.
+```
+
+**Related Topics**
+
+```text
+1. String
+```
+
+## :memo: Note

```

Comment:
This line should come with a markdownlint warning.
But it's OK, I'd add the markdownlint as a recommended vscode extension in my next commit.

Reply:
https://github.com/DavidAnson/markdownlint

Reply:
https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint